assets/js/googlesitekit/widgets/components/WidgetNull.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * WidgetNull component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import useWidgetStateEffect from '../hooks/useWidgetStateEffect';
+import Null from '../../../components/Null';
+
+// The supported props must match `Null` (except `widgetSlug`).
+export default function WidgetNull( { widgetSlug } ) {
+	useWidgetStateEffect( widgetSlug, Null, null );
```

Comment:
Per the ACs, this should set an empty object as `metadata` instead of `null`: Essentially, `metadata` should be directly providable as props (see usage in `WidgetAreaRenderer`), so `null` is not good for use here. Let's just make sure we rely on `useMemo` (like in the other similar components relying on `useWidgetStateEffect`) or a hard-coded empty object, to make sure it doesn't always set a _new_ (different) object.

Reply:
Updated, https://github.com/google/site-kit-wp/pull/2731/commits/a5596274fdf6b19c25a32e3f99b55cdab1eff061 hard-coded empty object makes more sense.